itest/rust/src/object_tests/base_init_test.rs

```diff
@@ -197,3 +197,78 @@ fn base_postinit_refcounted() -> TaskHandle {
     assert_eq!(obj.get_reference_count(), 2);
     next_frame(move || assert_eq!(obj.get_reference_count(), 1, "eventual dec-ref happens"))
 }
+
+fn make_mesh_arrays() -> Array<Variant> {
+    let mut arrays = Array::new();
+    arrays.resize(
+        mesh::ArrayType::MAX.ord().try_into().unwrap(),
+        &Variant::nil(),
+    );
+    let indices = PackedArray::<i32>::from([0, 1, 2]);
+    let vertex = PackedArray::<Vector2>::from([
+        Vector2::new(0.0, 0.0),
+        Vector2::new(1.0, 0.0),
+        Vector2::new(0.0, 1.0),
+    ]);
+    arrays.set(
+        mesh::ArrayType::INDEX.ord().try_into().unwrap(),
+        &indices.to_variant(),
+    );
+    arrays.set(
+        mesh::ArrayType::VERTEX.ord().try_into().unwrap(),
+        &vertex.to_variant(),
+    );
```

Comment:
You can use [`IndexEnum::to_index()`](https://godot-rust.github.io/docs/gdext/master/godot/classes/mesh/struct.ArrayType.html#impl-IndexEnum-for-ArrayType) here:
```suggestion
    arrays.set(
        mesh::ArrayType::INDEX.to_index(),
        &indices.to_variant(),
    );
    arrays.set(
        mesh::ArrayType::VERTEX.to_index(),
        &vertex.to_variant(),
    );
```